# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/strace/mpers-wrapper-g.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- strace-5.12/src/mpers.sh.vanilla	2021-05-06 18:36:04.911571634 +0200
+++ strace-5.12/src/mpers.sh	2021-05-06 18:36:32.293570543 +0200
@@ -6,6 +6,9 @@
 #
 # SPDX-License-Identifier: LGPL-2.1-or-later

+# our wrapper would remove -g* this scripts rely on
+export GCC_WRAPPER_BYPASS=1
+
 [ "x${D:-0}" != x1 ] || set -x
 
 export LC_ALL=C
